amenity-points.mss

```diff
@@ -27,8 +27,20 @@
     point-placement: interior;
   }
 
+  [feature = 'tourism_wilderness_hut'][zoom >= 13],
+  [feature = 'amenity_shelter']['shelter_type' = 'basic_hut'][zoom >= 14],
   [feature = 'amenity_shelter'][zoom >= 16] {
     marker-file: url('symbols/shelter-14.svg');
+    [feature = 'tourism_wilderness_hut'] {
+    	marker-file: url('symbols/wilderness_hut.svg');
+    	marker-transform:"scale(0.6)";
+    	[zoom < 19] { marker-transform:"scale(0.5)"; }
```

Comment:
Have the first lines be the conditions at low zoom, and then override for `zoom >= 19` if needed